### PR TITLE
NEXT-12744 - Add customize modules

### PR DESCRIPTION
**Pull request by raknison**
What should be done?
Create a new article on our GitBook instance which explains how to customize an existing module in the administration.
This article should mention:
The prerequisite, a working plugin (Refer to the plugin base guide)
Every other prerequisite you figure out during writing the guide (e.g. a subscriber, knowing how to create a service, a controller, etc.)
A short code example, including an explanation, on how to do it
Category: Extensions > Plugins > Administration

### CODE REVIEW

File: guides/plugins/plugins/administration/customizing-modules.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance. Of course, you'll have to understand JavaScript and have a basic familiarity with TwigJS, the templating engine, used in the Administration. However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Customizing a module
+
+Module settings like `color`, `icon`, `navigation` are fixed by design and cannot be changed.
+
+A guide for customizing components which are already defined in existing modules can be found here - [Customizing components](./customizing-components.md).
```
Comment (PaddyS) [inline guides/plugins/plugins/administration/customizing-modules.md:16]:
```suggestion
A guide for customizing components, which are already defined in existing modules, can be found here - [Customizing components](./customizing-components.md).
```

File: guides/plugins/plugins/administration/customizing-modules.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+At some point you need add or change the routes of a module. For example when you want to add a tab to the page
+
+This is done by creating a new component an implementing a `routeMiddleware`.
```
Comment (PaddyS) [inline guides/plugins/plugins/administration/customizing-modules.md:20]:
```suggestion
This is done by creating a new module and implementing a `routeMiddleware`.
```

File: guides/plugins/plugins/administration/customizing-modules.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+All you need for this guide is a running Shopware 6 instance. Of course, you'll have to understand JavaScript and have a basic familiarity with TwigJS, the templating engine, used in the Administration. However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Customizing a module
+
+Module settings like `color`, `icon`, `navigation` are fixed by design and cannot be changed.
+
+A guide for customizing components which are already defined in existing modules can be found here - [Customizing components](./customizing-components.md).
+
+At some point you need add or change the routes of a module. For example when you want to add a tab to the page
+
```
Comment (PaddyS) [inline guides/plugins/plugins/administration/customizing-modules.md:19]: Missing something like: "However, modules themselves cannot be directly overriden." or smth like that.

File: guides/plugins/plugins/administration/customizing-modules.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+## Next steps
+
+As you might have noticed, we are just adding a custom tab to the module. However, there's a lot more possible when it comes to extending the Adminsitration. You may want to try the following things:
+
+* If you want to learn about other possibilities to extend the Administration, we got you covered with other guides:
+    * [Writing templates](./writing-templates.md)
+    * [Add custom module](./add-custom-module.md)
+    * [Add menu entry](./add-menu-entry.md)
+    * [Add custom routes](./add-custom-route.md)
```
Comment (PaddyS) [inline guides/plugins/plugins/administration/customizing-modules.md:56]:
```suggestion
* [Writing templates](./writing-templates.md)
* [Add custom module](./add-custom-module.md)
* [Add menu entry](./add-menu-entry.md)
* [Add custom routes](./add-custom-route.md)
```

**Review by PaddyS — CHANGES_REQUESTED** [review]

File: guides/plugins/plugins/administration/customizing-modules.md (hunk 3 above)
Comment (raknison) [reply guides/plugins/plugins/administration/customizing-modules.md:19]: added your suggestion

**Review by leichteckig — APPROVED** [review]

**Pull request merged**